src/GraphQl/Subscription/SubscriptionManagerInterface.php

```diff
@@ -22,5 +22,5 @@ interface SubscriptionManagerInterface
 {
     public function retrieveSubscriptionId(array $context, ?array $result): ?string;
 
-    public function getPushPayloads(object $object): array;
+    public function getPushPayloads(object $object, string $type): array;
```

Comment:
this is a public interface, the added argument should be optional to cover the BC layer

Reply:
Ok